pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/DefaultRebalancePreChecker.java

```diff
@@ -227,9 +228,31 @@ private RebalancePreCheckerResult checkIsMinimizeDataMovement(String rebalanceJo
         return RebalancePreCheckerResult.warn(
             "minimizeDataMovement may not be enabled for consuming or completed, but instance assigment is allowed "
```

Comment:
nit: not a mistake in your PR, but can you fix spelling `assigment` to `assignment`